native/src/main/java/io/ballerina/stdlib/websocket/observability/WebSocketObserverContext.java

```diff
@@ -32,6 +32,7 @@ public class WebSocketObserverContext extends ObserverContext {
 
     private String connectionId = WebSocketObservabilityConstants.UNKNOWN;
     private String servicePathOrClientUrl = WebSocketObservabilityConstants.UNKNOWN;
+    private static final String protocol = "ws";
```

Comment:
Lets move the static variable to the top as it is the common practice.

Reply:
@ayeshLK address review suggestions